[docs] Fix some more grammar mistakes (#609)
gitpod-io/website: src/docs/index.md

Whether you just want to hack, have code to review or feel like trying something new on
GitHub, Gitpod launches a ready-to-code dev environment with a single click.

Because Gitpod is based on OSS tech like VS Code, Docker, and Kubernetes, it is familiar,
comprehensive, extensible, and easy to use. With deep GitHub integration, tools for
sharing, and a focus on usability, Gitpod contains your entire dev workflow in a
browser tab.

![](./images/gitpod-prefix.png)

> Unlike traditional cloud and desktop IDEs, Gitpod understands the context and prepares the

Reviewer: ```suggestion
Unlike traditional cloud and desktop IDEs, Gitpod understands the context and adjusts the
```

> IDE automatically. For instance, if you're creating a Gitpod workspace from a pull request,

Reviewer: ```suggestion
IDE automatically. For example, if you create a Gitpod workspace from a Pull Request,
```

> Gitpod will open in code-review mode.

Reviewer: ```suggestion
Gitpod will open in code-review mode.
```
